Source/Scene/Cesium3DTileFeature.js

```diff
@@ -129,6 +132,37 @@ define([
             }
         },
 
+        /**
+         * Gets and sets the material for the anchor line.
+         *
+         * @memberof Cesium3DTileFeature.prototype
+         *
+         * @type {Material}
+         *
+         * @default {@link Material.ColorType}
+         */
+        material : {
```

Comment:
Instead of `material`, why not `anchorLineColor`? `Polyline`s have a color material by default. You can get/set `polyline.material.uniforms.color`. If a polyline does not exist, return `undefined` or do nothing.